dso/model.py

```diff
@@ -33,6 +47,7 @@ class ScanArtifact:
 class Datasource:
     ARTEFACT_ENUMERATOR = 'artefact-enumerator'
     BDBA = 'bdba'
+    SAST_LINT_CHECK = 'sastLintCheck'
```

Comment:
nit: for reasons of consistency, preferably use "kebap-case" (dash-separated) rather than snake-case